jx_sqlite/setop_table.py

```diff
@@ -324,8 +342,19 @@ def _accumulate_nested(rows, row, nested_doc_details, parent_doc_id, parent_id_c
                             else:           # FACT IS EXPECTED TO BE A SINGLE VALUE, NOT AN OBJECT
                                 relative_path="."
 
-                            if relative_path == ".":
+                            if relative_path == "." and doc is None:
                                 doc = nested_value
+                            elif relative_path == "." and isinstance(nested_value, list):
+                                doc[push_name] = []
+                                for v in nested_value:
+                                    doc[push_name].append(v[push_name])
+                            elif relative_path == "." and isinstance(nested_value, dict):
+                                doc[push_name] = []
+                                for k, v in nested_value.items():
+                                    if k==push_name:
+                                        doc[push_name].append(v)
+                                    else:
```

Comment:
This block is also indicating there is a problem somewhere else:  Have `_accumulate_nested()` always return a list, and let the caller `unwraplist()`.